OpenRA.Game/Settings.cs

```diff
@@ -288,6 +288,19 @@ Hotkey GetKey(Func<KeySettings, Hotkey>[] keys, int index)
 		}
 	}
 
+	public class IrcSettings
+	{
+		public string[] Hostname = { "irc.openra.net" };
+		public int Port = 6667;
+		public string Nickname = "Newbie";
+		public string Username = "OpenRA";
```

Comment:
I think you are confused by IRC terminology here. The username is part of the ident which will be "~OpenRA@host86-159-185-244.range86-159.btcentralplus.com` when you connect with the in-game IRC client. It is usually configurable in IRC clients and not automatically updated with a nickname change.